stickynotes/stickynotes.c

```diff
@@ -50,978 +50,1055 @@ set_icon_geometry (GdkWindow *window,
                    int        width,
                    int        height)
 {
-	gulong data[4];
-	Display *dpy;
-
-	dpy = gdk_x11_display_get_xdisplay (gdk_window_get_display (window));
-
-	data[0] = x;
-	data[1] = y;
-	data[2] = width;
-	data[3] = height;
-
-	XChangeProperty (dpy,
-	                 GDK_WINDOW_XID (window),
-	                 gdk_x11_get_xatom_by_name_for_display (
-	                     gdk_window_get_display (window),
-	                     "_NET_WM_ICON_GEOMETRY"),
-	                 XA_CARDINAL, 32, PropModeReplace,
-	                 (guchar *)&data, 4);
+    gulong data[4];
+    Display *dpy;
+
+    dpy = gdk_x11_display_get_xdisplay (gdk_window_get_display (window));
+
+    data[0] = x;
+    data[1] = y;
+    data[2] = width;
+    data[3] = height;
+
+    XChangeProperty (dpy,
+                     GDK_WINDOW_XID (window),
+                     gdk_x11_get_xatom_by_name_for_display (
+                         gdk_window_get_display (window),
+                         "_NET_WM_ICON_GEOMETRY"),
+                     XA_CARDINAL, 32, PropModeReplace,
+                     (guchar *)&data, 4);
 }
 
 /* Called when a timeout occurs.  */
 static gboolean
 timeout_happened (gpointer data)
 {
-	if (GPOINTER_TO_UINT (data) == stickynotes->last_timeout_data)
-		stickynotes_save ();
-	return FALSE;
+    if (GPOINTER_TO_UINT (data) == stickynotes->last_timeout_data)
+        stickynotes_save ();
+        return FALSE;
```

Comment:
decrement the indent in this line

Reply:
The return doesn't belong to the if condition?
I will add a blank line to make it clear.
I hate this code-style. All was written without any space to the left or bottom.
.. and without using vars.....

Reply:
done.